src/status_im2/contexts/onboarding/welcome/views.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.onboarding.welcome.views
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im2.contexts.onboarding.welcome.style :as style]
+    [status-im2.contexts.onboarding.common.background :as background]))
+
+(defn page-title
+  []
+  [rn/view {:style style/title-container}
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/title-text-style)
+    (i18n/label :t/welcome-to-web3)]
+   [quo/text
+    (merge {:style {:color colors/white}}
+           style/subtitle-text-style)
+    (i18n/label :t/welcome-to-web3-sub-title)]])
+
+(defn welcome-button
+  []
+  [rn/view {:style style/welcome-buttons}
+   [quo/button
+    {:on-press                  #(rf/dispatch [:init-root :shell-stack])
+     :type                      :primary
+     :accessibility-label       :welcome-button
+     :override-background-color (colors/custom-color :magenta 60)}
+    (i18n/label :t/start-using-status)]])
+
+(defn navigate-back
+  [root]
+  [rn/view {:margin-top (if platform/ios? 44 0)}
+   [quo/page-nav
+    (merge {:horizontal-description? false
```

Comment:
this looks like too many params for simple back button, we should revisit page-nav component

Reply:
not in this PR